config/jobs/kubernetes/sig-azure/sig-azure-config.yaml

```diff
@@ -69,7 +69,55 @@ presubmits:
+        - "--test_args=--ginkgo.focus=\\[Conformance\\]|\\[NodeConformance\\] --ginkgo.skip=\\\
```

Comment:
this is a presubmit, I would probably use this job as a canary for whether splitting things up over a line is possible